lib/commands/list-subscriptions.js

```diff
@@ -21,28 +21,40 @@ module.exports = async (req, res, next) => {
     where: { channelId: command.channel_id },
   });
 
-  let repositories = await Promise.all(subscriptions.map(async (subscription) => {
+  let resources = await Promise.all(subscriptions.map(async (subscription) => {
     const github = await robot.auth(subscription.Installation.githubId);
+    let githubResource;
+
     try {
-      const repository = await github.repos.getById({ id: subscription.githubId });
-      return repository.data;
+      githubResource = await github.repos.getById({ id: subscription.githubId });
+      return githubResource.data;
     } catch (err) {
-      req.log.error({ err, repoId: subscription.githubId }, 'Could not find repository for subscription');
       if (err.code !== 404) {
         throw err;
       }
     }
+
+    try {
+      githubResource = await github.users.getById({ id: subscription.githubId });
+      return githubResource.data;
+    } catch (err) {
+      if (err.code !== 404) {
+        throw err;
+      }
+    }
+
```

Comment:
Rather than trying for both repository id and user id in this file, could we add another column in the `Subscriptions` table that indicates whether the subscription is to an "acccount" or to a "repo"?